tools/git/git-pr

```diff
@@ -152,7 +152,7 @@ fi
 currentBranch=$(git branch | grep "^*" | sed -e "s/^[*] //")
 if [ "${prDestinationBranch}" != "${currentBranch}" ] && [ ${force} -lt 1 ]; then
   echo "ERROR: This PR is made against branch '${prDestinationBranch}' while your current checked out branch is '${currentBranch}'."
-  echo "ERROR: Please make sure you're in the right branch and run this scipt again."
+  echo "ERROR: Please make sure you're in the right branch and run this script again."
```

Comment:
this change prevents the regular PR change process from working. I suggest pushing this change separately (outside of this PR)

Reply:
@jbampton can you exclude this change please? I am going to risk comiting this on main directly as it is only a text change.